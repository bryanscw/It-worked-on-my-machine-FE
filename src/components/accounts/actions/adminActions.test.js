import React from 'react'
import { render, cleanup } from '@testing-library/react'
import fetchUsers, { createUser } from './adminActions.js';

import updateUser from './adminActions.js';
import deleteUser from './adminActions.js';


afterEach(cleanup);

it('should take a snapshot', () => {
    const { asFragment } = render(<fetchUsers />);
    expect(asFragment()).toMatchSnapshot();
});

it('should take a snapshot', () => {
    const { asFragment } = render(<createUser />);
    expect(asFragment()).toMatchSnapshot();
});

it('should take a snapshot', () => {
    const { asFragment } = render(<updateUser />);
    expect(asFragment()).toMatchSnapshot();
});

it('should take a snapshot', () => {
    const { asFragment } = render(<deleteUser />);
    expect(asFragment()).toMatchSnapshot();
});