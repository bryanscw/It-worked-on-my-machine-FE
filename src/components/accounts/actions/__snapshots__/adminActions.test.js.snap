// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <fetchusers />
</DocumentFragment>
`;

exports[`should take a snapshot 2`] = `
<DocumentFragment>
  <createuser />
</DocumentFragment>
`;

exports[`should take a snapshot 3`] = `
<DocumentFragment>
  <updateuser />
</DocumentFragment>
`;

exports[`should take a snapshot 4`] = `
<DocumentFragment>
  <deleteuser />
</DocumentFragment>
`;
